news/compute_selection_fixes.rst

```diff
@@ -0,0 +1,27 @@
+**Added:**
+
+* OpenMMEngineSettings now has a `gpu_device_index` attribute
+  allowing users to pass through a list of ints to select the
+  GPU devices to run their simulations on.
+
+**Changed:**
+
+* `openfe.protocols.openmm_rfe._rfe_utils.compute` has been moved
```

Comment:
good example of a nice changelog entry but since this was a private API, no symver major bump needed